### PR TITLE
Issue34_Home_AboutUs_EAMenu_updates

### PR DESCRIPTION
**Pull request by vricci518**
#34 
Assigned Bootstrap to Home and AboutUs pages
Started to assign Bootstrap to EA Menu page

BUG ISSUES
Home page issues with element in reservation, curbside and gift cards. Elements switch to 1 across at iPad screen size but revert to 2 across at mobile phone size.
Common footer does not extend width of home page

### CODE REVIEW

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
Gotta fix conflicts first

**Review by margeschrec — APPROVED** [review]

**Review by billythakidd04 — DISMISSED** [review]

File: aboutus.php [hunk 1]
```diff
@@ -51,9 +51,9 @@
 				</div>
 			</div>
 		</nav>
-	</header>
-    <div class="mainabout">
-        <div class="about">
+	</header><br>
```
Comment (billythakidd04) [inline aboutus.php]: close the br tags

File: css/home.css [hunk 2]
```diff
@@ -20,17 +13,20 @@
 }
 .reservation h3{
     text-align: center;
+    margin: auto 5%;
 }
-.location{
+.col{
```
Comment (billythakidd04) [inline css/home.css]: I would recommend not doing this and use https://getbootstrap.com/docs/5.0/utilities/text/#text-alignment instead

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
2 changes and remove the .DS_Store file

File: aboutus.php [hunk 3]
```diff
@@ -4,56 +4,57 @@
...
+                                <li><a href="aboutus.php">About Us</a></li>
+                                <li><a href="faq.php">FAQ</a></li>
+                                <li><a href="contact.php">Contact Us</a></li>
+                            </ul>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </header><br>
```
Comment (billythakidd04) [inline aboutus.php]: you _shouldn't_ need a `<br/>` there but if you do please make sure it's closed

File: home.php [hunk 4]
```diff
@@ -53,63 +53,91 @@
...
 		</nav>
 	</header>
-	<div>
-		<div name="home_page_photo">
-			<img src="images/barbilltavern.jpg" id="homephoto" alt="bar bill tavern">
+	<div class="container-fluid">
+		<div id="carouselExampleSlidesOnly" class="carousel slide" data-bs-ride="carousel">
+			<div class="carousel-inner">
+				<div class="carousel-item active">
+					<img src="images/caro_bbt.jpeg" class="d-block w-100" alt="barbill tavern">
```
Comment (billythakidd04) [inline home.php]: close `` tags

File: home.php [hunk 5]
```diff
@@ -53,63 +53,91 @@
...
+				</div>
+				<div class="carousel-item">
+					<img src="images/caro_wings.jpeg" class="d-block w-100" alt="chicken wings">
```
Comment (billythakidd04) [inline home.php]: close tag

File: home.php [hunk 6]
```diff
@@ -53,63 +53,91 @@
...
+				</div>
+				<div class="carousel-item">
+					<img src="images/caro_rbeef.jpeg" class="d-block w-100" alt="roastbeef sandwich">
```
Comment (billythakidd04) [inline home.php]: close tag

File: home.php [hunk 7]
```diff
@@ -53,63 +53,91 @@
...
+				</div>
+			</div>
 		</div>
-		<div class="reservation">
+	</div>
+	<br />
```
Comment (billythakidd04) [inline home.php:71]: using bootstrap means this shouldn't be needed either...

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
Still have to remove the .DS_Store file and I'm curious about the br tags. They shouldnt be needed. If they are, so be it but please make sure tags are closed.

**Review by billythakidd04 — APPROVED** [review]

**Pull request merged**